views/js/test/assets/strategies/test.js

```diff
@@ -11,6 +11,45 @@ define([
         assert.ok(typeof strategies === 'object', "The module exports an object");
     });
 
+    QUnit.module('External Replaced strategy');
+
+    QUnit.test('expected strategy format', 3, function(assert){
+
+        assert.ok(typeof strategies.externalReplaced === 'object', "The external replaced strategy exists");
+        assert.equal(strategies.externalReplaced.name, 'externalReplaced', "The external replaced strategy has the right name");
+        assert.ok(typeof strategies.externalReplaced.handle === 'function', "The external replaced strategy has an handler");
+    });
+
+    var externalReplacedDataProvider = [{
```

Comment:
Could you please add the following cases : 
 - ` assets   : {'img' : {'test.png': 'http://my.cdn/my/test.png?signed=true' } } `
 - ` assets   : {'img' : {'test.png': 'test.png' } } `
 - ` assets   : {'img' : {'http://my.cdn/my/test.png': 'test.png' } } `